src/main/java/com/enterprise/fundraisermanager/dao/FundraiserDAOStub.java

```diff
@@ -1,5 +1,12 @@
 package com.enterprise.fundraisermanager.dao;
 
```

Comment:
Added a "title" documentation comment to each file
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
These are pretty much deprecated tags, now that we can see commit history in version control.
Believe it or not, that used to not be so easy to see. 
I'd remove this, or replace it with a descriptive JavaDoc, as it is not adding value.